zappa/cli.py

```diff
@@ -1716,7 +1716,7 @@ def init(self, settings_file="zappa_settings.json"):
             env: {
                 'profile_name': profile_name,
                 's3_bucket': bucket,
-                'runtime': 'python3.6' if sys.version_info[0] == 3 else 'python2.7',
+                'runtime': get_venv_from_python_version(),
```

Comment:
I think it's better to use `get_runtime_from_python_version`.
Because:
previous code allows only `'python2.7'` and `'python3.6'`
suggested code allows any `pythonX.X` according to `sys.version`

```suggestion
                'runtime': get_runtime_from_python_version,
```

Reply:
doesn't `get_runtime_from_python_version()` only return `python2.7` or `python3.6`?
https://github.com/Miserlou/Zappa/blob/6a43ed0b06e5877e71e829686fad93f8e51b78c1/zappa/utilities.py#L173 seems to be the better way as it gets the version from `*sys.version_info`

Reply:
@itamt Because of https://github.com/Miserlou/Zappa/blob/6a43ed0b06e5877e71e829686fad93f8e51b78c1/zappa/__init__.py#L3 and the exception that is raised if you run CLI using anything other than one of the supported versions, both functions [get_venv_from_python_version](https://github.com/Miserlou/Zappa/blob/6a43ed0b06e5877e71e829686fad93f8e51b78c1/zappa/utilities.py#L173) and [get_runtime_from_python_version](https://github.com/Miserlou/Zappa/blob/6a43ed0b06e5877e71e829686fad93f8e51b78c1/zappa/utilities.py#L176) will essentially produce identical output.

In my view, relying on `get_venv_from_python_version` is more sustainable in the long run as new python versions are enabled in the future and less code changes are needed to support them. However, as the code is right now, it does not matter which function is used.

As far as this PR is concerned, the proposed change here is to move from hand crafted logic `'python3.6' if sys.version_info[0] == 3 else 'python2.7'` to an existing function that produces similar output. There is further room to cleanup use of both functions and make the code more pythonic ( [There should be one-- and preferably only one --obvious way to do it.](https://www.python.org/dev/peps/pep-0020/) ), however, such changes are outside the scope of this PR.